perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -117,7 +116,7 @@ def _Create(self):
       memory = self.spec.vm_spec.memory
       cpus = self.spec.vm_spec.cpus
       self._ValidateMachineType(memory, cpus)
```

Comment:
If the user of this benchmark forgets to specify memory and cpu in vm_spec, we should identify that error and output a more specific error message such as: "for GCP postgres instances, you must specify specific memory and CPU values instead of machine type".

If we just let _ValidateMachineType(), it would throw "some" sort of error but it would be hard for the user of PKB to figure out exactly what went wrong, especially if they have been using this code to benchmark MySQL instance where they'd only need to specify machine_type.